db/migrations/versions/006_replace_enums_with_strings.py

```diff
@@ -0,0 +1,232 @@
+"""empty message
+
+Revision ID: 006_replace_enums_with_strings.py
+Revises: 005_relax_proj_prog_nulls
+Create Date: 2023-07-03 12:10:30.753706
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "006_replace_enums_with_strings.py"
```

Comment:
`006_replace_enums_with_strings.py` should be `006_replace_enums_with_strings`

Reply:
done